tests/config/test_data.config

```diff
@@ -662,8 +662,9 @@ params {
             'parameter' {
                 maxquant                        = "${params.test_data_base}/data/proteomics/parameter/mqpar.xml"
             }
-            'idfile' {
-                openms_idxml                    = "${params.test_data_base}/data/proteomics/openms_idxml/BSA_QC_file.idXML"
```

Comment:
Is removing this intentional? Aw we sure nobody is using it?

Reply:
Yes! :) I asked the person, who committed this -> https://github.com/nf-core/test-datasets/commit/f33fb62c57ae463c82ad6bbea6082516e9b81a8b#commitcomment-130295954
He never uploaded it on the testdata repo

Reply:
Ah yes and I double-checked that nobody is using it